utils/v4/docs/get_api_docs.sh

```
@@ -0,0 +1,16 @@
+# Get API documentation (public endpoint, no auth required)
+# Usage: ./get_api_docs.sh
+# API_URL=http://localhost:5001 ./get_api_docs.sh
+# API_URL=https://api-gw.dev.platform.linuxfoundation.org/cla-service ./get_api_docs.sh
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Handle API URL
+. ${SCRIPT_DIR}/../shared/handle_api_url.sh
+
+# Set up curl execution
+API="${API_URL}/v4/api-docs"
```

Shellcheck (0.11.0)

[warning] API appears unused. Verify use (or export if used externally).

(SC2034)